h/search/query.py

```diff
@@ -363,6 +363,17 @@ def __call__(self, search, _):
         return search.filter(Q("bool", should=should_clauses))
 
 
+class RepliesFilter:
+    """A filter to limit to replies to specified ids."""
+
+    def __call__(self, search, params):
+        # Pop the key, so it's not picked up by the `KeyValueMatcher`
```

Comment:
I don't think I quite get the whole pop-ing things so they are not present next in the chain in all this H PRs.

Here we just seem to want to avoid the "default" matcher to pick this one up again. 

I've no idea how many keys `KeyValueMatcher` handles.

Reply:
Neither do I. I think it's anything that a previous filter / handler hasn't grabbed it.